dotnet/src/IntegrationTests/Connectors/Memory/Postgres/PostgresMemoryStoreTests.cs

```diff
@@ -637,85 +504,181 @@ public async Task GetNearestMatchesDifferentiatesIdenticalVectorsByKeyAsync()
+    [Fact(Skip = SkipReason)]
     public async Task ItCanBatchRemoveRecordsAsync()
     {
         // Arrange
-        using PostgresMemoryStore db = await this.CreateMemoryStoreAsync();
+        PostgresMemoryStore memoryStore = this.CreateMemoryStore();
         int numRecords = 10;
-        string collection = "test_collection" + this._collectionNum;
-        this._collectionNum++;
+        string collection = "test_collection";
         IEnumerable<MemoryRecord> records = this.CreateBatchRecords(numRecords);
-        await db.CreateCollectionAsync(collection);
+        await memoryStore.CreateCollectionAsync(collection);
 
         List<string> keys = new();
 
         // Act
-        await foreach (var key in db.UpsertBatchAsync(collection, records))
+        await foreach (var key in memoryStore.UpsertBatchAsync(collection, records))
         {
             keys.Add(key);
         }
 
-        await db.RemoveBatchAsync(collection, keys);
+        await memoryStore.RemoveBatchAsync(collection, keys);
 
         // Assert
-        await foreach (var result in db.GetBatchAsync(collection, keys))
+        await foreach (var result in memoryStore.GetBatchAsync(collection, keys))
         {
             Assert.Null(result);
         }
     }
 
-    [Fact(Skip = SkipOrNot)]
+    [Fact(Skip = SkipReason)]
     public async Task DeletingNonExistentCollectionDoesNothingAsync()
     {
         // Arrange
-        using PostgresMemoryStore db = await this.CreateMemoryStoreAsync();
-        string collection = "test_collection" + this._collectionNum;
-        this._collectionNum++;
+        PostgresMemoryStore memoryStore = this.CreateMemoryStore();
+        string collection = "test_collection";
+
+        // Act
+        await memoryStore.DeleteCollectionAsync(collection);
+    }
+
+    [Fact(Skip = SkipReason)]
+    public async Task ItCanBatchGetRecordsAndSkipIfKeysDoNotExistAsync()
+    {
+        // Arrange
+        PostgresMemoryStore memoryStore = this.CreateMemoryStore();
+        int numRecords = 10;
+        string collection = "test_collection";
+        IEnumerable<MemoryRecord> records = this.CreateBatchRecords(numRecords);
 
         // Act
-        await db.DeleteCollectionAsync(collection);
+        await memoryStore.CreateCollectionAsync(collection);
+        var keys = await memoryStore.UpsertBatchAsync(collection, records).ToListAsync();
+        keys.Insert(0, "not-exist-key-0");
+        keys.Insert(5, "not-exist-key-5");
+        keys.Add("not-exist-key-n");
+        var resultRecords = memoryStore.GetBatchAsync(collection, keys);
+
+        // Assert
+        Assert.NotNull(keys);
+        Assert.Equal(numRecords, keys.Count - 3);
+        Assert.Equal(numRecords, resultRecords.ToEnumerable().Count());
+    }
+
+    #region private ================================================================================
+
+    private const string ConnectionString = "Host=localhost;Database={0};User Id=postgres";
```

Comment:
This connection string misses `Password` property, so it's not possible to run integration tests without modifying it. 

Please add section for Postgres in this configuration file with empty value:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/IntegrationTests/testsettings.json

Then read Postgres section and password like we do this for OpenAI for example:
https://github.com/microsoft/semantic-kernel/blob/16d52c155a2c8193fef7207a8816ac8c4a8eed2c/dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs#L32-L38

Later we have a plan to start Docker containers during build pipeline, so it will be possible to run all integration tests which currently rely on Docker and are ignored.

This change will allow to store password for Postgres together with other keys and secrets for all our integration tests.